JoyfillExample/JoyfillExample/JoyfillExampleApp.swift

```diff
@@ -12,7 +12,7 @@ import JoyfillAPIService
 struct JoyfillExampleApp: App {
     init() {
         JoyfillAPIService.initialize(
-            userAccessToken: "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ0b2tlbiI6IjY1Yzc2NDI5ZGQ5NjIwNmM3ZTA3ZWQ5YiJ9.OhI3aY3na-3f1WWND8y9zU8xXo4R0SIUSR2BLB3vbsk",
+            userAccessToken: "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ0b2tlbiI6IjY2MjhmMTIzNDg5MjYxOGZjMTE4NTc3MyJ9.JofMjGzwWc_nOUcxAyOpuoX_IWIERX7fWJILPda_7ys",
```

Comment:
We should leave this out and when they go to run the example we warn them if they have not provided one here, with also logging that message with the link to the our instructions on how to get an access token